overlord/swfeats/registry.go

```diff
@@ -0,0 +1,114 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2025 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// The swfeats package implements registries for features
+// not tracked in other places
+package swfeats
+
+import (
+	"fmt"
+	"strings"
+)
+
+var (
+	ChangeReg *ChangeKindRegistry = newChangeKindRegistry()
+	EnsureReg *EnsureRegistry     = newEnsureRegistry()
```

Comment:
I saw a similar comment about this from @andrewphelpsj, but I would consider exposing functions, e.g. `ChangeKindRegister(...)` rather than the registry objects